### DISCUSSION TITLE
Dumb question: Can I force FPS in the Home Assistant version?

### DISCUSSION THREAD

**Discussion by thatdaveguy1** [mrlt8/docker-wyze-bridge#630] [Q&A]
I assume I can I am just a dummy and can’t figure out the different home assistant syntax

**Comment by mrlt8** [comment]
Force FPS is slightly buggy in the current version, but will try to add an option for Home Assistant once I get around to updating the FPS to work when night mode is active.

**Reply by thatdaveguy1** [reply]
> Force FPS is slightly buggy in the current version, but will try to add an option for Home Assistant once I get around to updating the FPS to work when night mode is active.

Thank you! In my case, I don’t use night mode so if I had an option, it would work great from day one I think.
